### DISCUSSION TITLE
Support multi languages search

### DISCUSSION THREAD

**Discussion by whzy1314** [searxng/searxng#1573] [Ideas]
As a dual language speaker, it would be amazing if I could select multiple searching languages. Currently I will always have to switch languages and search again.  

It was originally brought it up here in original searx project.. but still remains open https://github.com/searx/searx/issues/1344

**Comment by mrbreil** [comment]
thumbs up

**Comment by leana8959** [comment]
Hello, I found your question and I found [this](https://docs.searxng.org/admin/settings/settings_engine.html#example-multilingual-search) right after. If you're admin of your own instance, this might help.
